README.md

```diff
@@ -40,29 +42,43 @@ libyui_client supports different types of widgets.
 Here are examples of usage:
 
 ### Button
-
 ```ruby
 app.button(id: 'test_id').click  # clicks the button with id 'test_id'
 app.button(label: 'test_label').debug_label # get 'debug_label' value with label 'test_label'
 ```
 
-### Checkbox
+### Bargraph
+```ruby
+# Get list of labels in the segments:
+app.bargraph(class: YBarGraph).labels
+# Get array of hashes containing label->value pairs denoting each segment:
+app.bargraph(class: YBarGraph).segments
+# Get list of values in the segments:
+app.bargraph(class: YBarGraph).values
+```
 
+### Checkbox
 ```ruby
 app.checkbox(id: 'test_id').check  # checks the checkbox with id 'test_id'
 app.checkbox(label: 'test_label').checked? # gets the state of checkbox with label 'test_label'
 app.checkbox(id: 'test_id').toggle # toggles the checkbox with id 'test_id'
 app.checkbox(id: 'test_id').uncheck  # unchecks the checkbox with id 'test_id'
-
 ```
 
 ### Combobox
-
 ```ruby
 app.combobox(id: 'test_id').items  # gets all available items for combobox with id 'test_id'
 app.combobox(id: 'test_id').select  # selects the checkbox with id 'test_id'
 app.combobox(id: 'test_id').selected_item # gets the selected item in combobox with id 'test_id'
+app.combobox(label: 'cmbx', class: 'YComboBox').set # Sets custom string to the editable checkbox
+```
+
+### Datefield
+```ruby
+# Set date field to 2048-08-16
+app.datefield(class: 'YDateField').set(Date.new(2048, 8, 16))
```

Comment:
I believe it's better to use Date object in the tests, because you can create DateTime with date and time and pass it to be set in the fields. For the string, api actually allows to claim field to be `textfield` and send string there.